packages/openapi3-parser/lib/parser/oas/parseParameterObject.js

```diff
@@ -166,6 +168,14 @@ function parseParameterObject(context, object) {
       const example = parameter.get('example');
       const member = new namespace.elements.Member(parameter.get('name'), example);
 
+      if (example === undefined) {
+        const schema = parameter.get('schema');
+
+        if (schema) {
+          member.value = schema;
+        }
+      }
+
       const description = parameter.get('description');
```

Comment:
@kylef Is the lack of setting `member.title` intentional in OAS parsers?

The `drafter` APIB parser seems to be [setting 'title' to the parameter type](https://github.com/apiaryio/drafter/blob/master/packages/drafter/src/RefractAPI.cc#L209)